src/docs/introduction/getting-started-with-ftw-daily/index.md

```diff
@@ -1,162 +1,125 @@
 ---
-title: Getting started with Flex Template for Web
-slug: getting-started-with-ftw-daily
-updated: 2021-12-14
+title: Getting started with Sharetribe Web Template
+slug: getting-started-with-web-template
+updated: 2023-03-07
 category: introduction-getting-started
 ingress:
-  Learn how to install one of the Flex Templates for Web (FTW) to your
-  local development environment.
+  Learn how to install Sharetribe Web Template to your local development
+  environment.
 skills: basic command line, text editing
 published: true
 ---
 
-Flex Templates for Web (FTW) are marketplace web applications built on
+The Sharetribe Web Template is a marketplace web application built on
 top of the
-[Marketplace API](/operator-guides/concepts/#marketplace-api). While you
-can create a marketplace purely using just the API, it requires a
-significant amount of effort (both money and time) and we recommened
-using the template as a starting point for customizations.
+[Marketplace API](/introduction/introducing-flex/#the-marketplace-api).
+While you can create a marketplace purely using just the API, it
+requires a significant amount of effort (both money and time) and we
+recommened using the template as a starting point for customizations.
 
-FTW-daily is built with [React](https://reactjs.org/),
+The Sharetribe Web Template is built with [React](https://reactjs.org/),
 [Redux](https://redux.js.org/), and
 [CSS Modules](https://github.com/css-modules/css-modules). It also
 contains a small [Node.js](https://nodejs.org/en/) server, which
-provides server-side rendering (SSR) for the production site.
+provides server-side rendering (SSR) for the deployed site.
 
-The purpose of this guide is to clone and configure FTW-daily to your
-local development environment - and then get it up and running. This
-guide also helps you to create accounts to Stripe and Mapbox. Those
-services are needed to run the FTW-daily template app.
-
-If you want to use FTW-hourly or FTW-product as your starting point, you
-can find the relevant options in this tutorial as well. However, if you
-are intending to work through the tutorial steps, we recommend starting
-with FTW-daily, as it is the basis for the tutorials.
+The purpose of this guide is to clone and configure the Sharetribe Web
+Template to your local development environment - and then get it up and
+running. This guide also helps you to create accounts to Stripe and
+Mapbox. Those services are needed to run the Sharetribe Web Template
+app.
 
 ## Setup development environment
 
 ### Prerequisities
```

Comment:
There should be a note or section that customizers need to first (check that the correct processes exist in the marketplace and) add the new processes if they don't exist.

I added something pretty rough to the template's README.md:
https://github.com/sharetribe/web-template#take-the-new-beta-processes-into-use

Reply:
Added!